src/formatDate.js

```diff
@@ -50,7 +50,44 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const result = date.split(/,|-|\.|\//);
```

Comment:
it will be better not to hardcode all possible separators for date but use the actual one from `fromFormat`: `fromFormat[fromFormat.length - 1]`